static/sass/styles.scss

```diff
@@ -451,7 +451,7 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
 }
 
 .p-equal-opportunities {
-  background-color: #f4f4f3;
+  background-color: #f3f3f3;
```

Comment:
Soon Vanilla with have `$color-paper`, maybe worth extracting this to variable locally as well?